Datasnap/Classes/Event Properties/Types.h

```diff
@@ -9,7 +9,7 @@
 #import "EventProperty.h"
 
 @interface Types : EventProperty
-@property NSString* id;
+@property NSString* identifier;
```

Comment:
@almcintyre, @DevinJohnson: if we are defining 'identifier' in multiple subclasses we should just put it in the 'EventProperty' superclass